github-dark.css

```diff
@@ -2924,9 +2927,21 @@
   .bot-identifier, .dashboard-notice .dismiss:hover, .text-gray, a.text-gray,
   td.icon .octicon-file-text, td.icon .octicon-file-symlink-file,
   .UnderlineNav-item, a.UnderlineNav-item, .project-header-link,
-  a.project-header-link, button.project-header-link, td.icon {
+  a.project-header-link, button.project-header-link, td.icon,
+  .js-hide-project-card-details, .js-hide-project-menu {
     color: #949494 !important;
   }
+  /* https://github.com/StylishThemes/GitHub-Dark/issues/640 */
+  .js-hide-project-card-details:hover, .js-hide-project-menu:hover {
```

Comment:
Thanks, Im using it because Ive been previously instructed to use a similar class on another fix. Maybe related or not IDK.

See https://github.com/StylishThemes/GitHub-Dark/pull/636#issuecomment-397075091

Ill wait for @Mottie to give feedback and then we'll see :)

Reply:
Yeah @xt0rted is right, I should have avoiding using the `.js-` selector. Please update it.

Reply:
Also, combine this with the already existing `color: #c0c0c0 !important;` entry.

Reply:
thx gimme a bit